docker-compose-jetson.yaml

```diff
@@ -35,6 +35,8 @@ services:
       ROS_MASTER_URI: http://${MASTER_IP:-127.0.0.1}:11311
       GAZEBO_MASTER_URI: http://${MASTER_IP:-127.0.0.1}:11345
       CYCLONEDDS_URI: ${CYCLONEDDS_URI}
+      CYCLONEDDS_NETWORK_INTERFACE_NAME: ${CYCLONEDDS_NETWORK_INTERFACE_NAME}
+      CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE: ${CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE}
```

Comment:
What happens if you do not specify `CYCLONEDDS_NETWORK_INTERFACE_AUTODETERMINE` in Jetson? You may want to specify a default value.

Reply:
The default value should be set. I updated the file.